blueflood-core/src/main/java/com/rackspacecloud/blueflood/rollup/SlotKey.java

```diff
@@ -135,6 +136,33 @@ public static SlotKey parse(String string) {
     }
 
     /**
+     * Same as the method {@link #getChildrenKeys()} except this method returns only
+     * the children corresponding to the destination granularity.
+     *
+     * For example, a slot key of granularity
+     * {@link Granularity#MIN_1440 MIN_1440}, for destination granularity of
+     * {@link Granularity#MIN_60 MIN_60}, will have 6*4=24 children.
```

Comment:
Curious, Is this a normal example? I mean, do we normally ask from 1 day (1440m) granularity to 1 hr (60m) granularity, skipping the 240m granularity? 

Also, the term "children" is not as clear, but I realized it's probably been used in other places. At first I thought 1440m is a child of the 240m. I don't know if it's possible to change to "Finer" and "Coarser"